components/dashboard/src/projects/Project.tsx

```diff
@@ -130,8 +105,12 @@ export default function () {
         history.push(`/${team?.slug}/${projectName}/${pb.id}`);
     }
 
+    const formatDate = (date: string | undefined) => {
+        return date ? moment(date).fromNow() : "";
+    }
+
     return <>
-        <Header title={project?.name || ""} subtitle={toRemoteURL(project?.cloneUrl || "")} />
+        <Header title={"Overview"} subtitle={<h2>View all active branches for <strong>{project?.name}</strong></h2>} />
```

Comment:
**suggestion**: We're now missing a link to the repository, could we add back the link to the repository? Using `text-gray-600` on hover for the link could suffice. Also, let's bring back the trailing dot for consistency with other headers. 🤓 

```suggestion
        <Header title={"Overview"} subtitle={<h2>View all active branches for <strong>{project?.name}</strong>.</h2>} />
```